web/static/webcomponents/pages/ApiKeys/ApiKeys-home/ApiKeys-home.js

```diff
@@ -0,0 +1,107 @@
+import { fromHTML } from "../../../utils.js";
+
+const apikeysHomeTemplate = document.createElement("template");
+apikeysHomeTemplate.innerHTML = `
+<link rel="stylesheet" type="text/css" href="/static/reset.css">
+<link rel="stylesheet" type="text/css" href="/static/webcomponents/pages/apikeys/apikeys-home/apikeys-home.css">
+`;
+
+class ApiKeysHome extends HTMLElement {
+  constructor() {
+    super();
+    this.shadow = this.attachShadow({ mode: "open" });
+    this.shadow.append(apikeysHomeTemplate.content.cloneNode(true));
+    this.apiKeysData = [];
+  }
+
+  async fetchApiKeys() {
+    try {
+      const response = await fetch("/api/v1/apikeys");
+      if (!response.ok) {
+        throw new Error(`HTTP error! Status: ${response.status}`);
+      }
+      const data = await response.json();
+      this.apiKeysData = data.data.apiKeys.reverse(); // Reverse the order of the API keys
```

Comment:
Once you flatten the structure above, I think you can reduce the nesting here to go along with that fix.

Reply:
Change this to `data.data.apiKeys` => `data.data`